Website/src/rendering/src/assets/css/components/Products/ProductDetails.css

```diff
@@ -12,7 +12,7 @@
 
         .image-active {
           > div {
-            @apply relative pt-[100%];
+            @apply relative pt-[100%] border-2 border-gray;
```

Comment:
This should be removed because it adds a border to the product image, not the thumbnail.

<img width="484" alt="image" src="https://user-images.githubusercontent.com/96472119/164524012-fa464ee0-e26a-4042-a126-897340580cc4.png">